include/openssl/e_os2.h

```diff
@@ -276,6 +276,18 @@ typedef unsigned __int64 uint64_t;
 #  endif
 # endif
 
+/* Format specifier for printing size_t */
+# if (defined(__STDC_VERSION__) && __STDC_VERSION__ >= 199901L) \
+      && !defined(__VMS)
+#  define OSSLzu  "zu"
+# else
+#  ifdef SIXTY_FOUR_BIT_LONG
+#   define OSSLzu PRIu64
```

Comment:
Could it be that the `#else` part should look like this?

``` C
# else
#  ifdef THIRTY_TWO_BIT
#   define OSSLzu "lu"
#  else
#   define OSSLzu PRIu64
#  endif
# endif
```

Note that `THIRTY_TWO_BIT` is the default unless we say otherwise in the config targets.